clang/docs/LibASTMatchersTutorial.rst

```diff
@@ -22,8 +22,8 @@ started guide <https://llvm.org/docs/GettingStarted.html>`_.
 
 .. code-block:: console
 
-      cd ~/clang-llvm
-      git clone https://github.com/llvm/llvm-project.git
+      mkdir ~/clang-llvm && cd ~/clang-llvm
+      git clone https://github.com/llvm/llvm-project.git .
```

Comment:
Given the:
```
      cd ~/clang-llvm
      git clone git://cmake.org/stage/cmake.git
```
this isn't a good idea. Better to clone it to an llvm-project subdirectory (i.e. drop the .) and alter later steps.

Reply:
Thanks for the review @jrtc27. Indeed, having the `llvm-project` subdirectory is much cleaner. I updated all required steps. Found some more outdated details. Please re-review. Thank you.